<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>reveal.js</title>

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/black.css">

		<!-- Theme used for syntax highlighting of code -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<section>
					<h4>Carnegie Mellon Ranked First in publications in highly-selective Computational Biology conferences</h4>
					<p style="font-size:20px;">Oct 18, 2017</p>
					<div style="width:50%; float:left;">
						<img src="images/slide1.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Based on analysis by csrankings.org, Carnegie Mellon is the front-runner in computational biology. The rankings are based upon accepted proceedings papers at the top computational biology conferences, ISMB and RECOMB.  These conferences typically accept only around 15% of submitted papers and the number of accepted papers over many years is a strong indication of Carnegie Mellon’s leadership in framing novel biomedical research questions in computational terms and developing rigorous and scalable solutions. Carnegie Mellon was also ranked first across all of computer science, consistent with many other ranking metrics.</p>
					</div>
				</section>
				<section>
					<h4>Computational Biology Guarantees Research Experience for Undergrads</h4>
					<p style="font-size:20px;">Oct 30, 2017</p>
					<div style="width:50%; float:left;">
						<img src="images/slide2.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Doing undergraduate research for credit has long been an option exercised at Carnegie Mellon and other universities, but the Computational Biology Department has gone a step further and is guaranteeing that all students in its new undergraduate degree program will be able to pursue a semester-long research experience should they wish to.“Our faculty want to make it a central feature of our program,” said Phillip Compeau, assistant department head for education. “We believe this will be a way to attract students,” he added, noting computational biology majors may be particularly interested in research compared to other students in the School of Computer Science.</p>
					</div>
				</section>
				<section>
					<h4>CMU Software Assembles RNA Transcripts More Accurately</h4>
					<p style="font-size:20px;">Nov 13, 2017</p>
					<div style="width:50%; float:left;">
						<img src="images/slide3.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Method should help scientists understand regulation of gene expression.</p>
					</div>
				</section>
				<section>
					<h4>The Computational Biology Department Undergraduate Program Welcomes Two New Students</h4>
					<p style="font-size:20px;">November 14, 2017</p>
					<table>
						<tr>
							<td style="margin:auto; border:none; align-content: center;"><img src="images/slide4a.png" style="width:50%; margin:auto;" /></td>
							<td style="margin:auto; border:none; align-content: center;"><img src="images/slide4b.png" style="width:50%; margin:auto;" /></td>
						</tr>
						<tr>
							<p style="font-size:20px;">Welcome Doug and Wendy! The Computational Biology Department will be welcoming its first freshman class of School of Computer Science students in the spring.</p>
						</tr>
					</table>
				</section>
				<section>
					<h4>Natalie Sauerwald awarded Richard King Mellon Foundation Presidential Scholarship in Life Sciences</h4>
					<p style="font-size:20px;">Dec 5, 2017</p>
					<div style="width:50%; float:left;">
						<img src="images/slide5.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Natalie Sauerwald, a 3rd year Ph.D student, has been awarded the Richard King Mellon Foundation Presidential Scholarship in the Life Sciences.  She is a member of Dr. Carl Kingsford’s lab.  Congratulations, Natalie!</p>
					</div>
				</section>
				<section>
					<h4>Bar-Joseph Appointed to Endowed Chair</h4>
					<p style="font-size:20px;">Dec 8, 2017 </p>
					<div style="width:50%; float:left;">
						<img src="images/slide6.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:18px;">At a ceremony this week, Professor Ziv Bar-Joseph was appointed FORE Systems Professor of Computational Biology and Machine Learning.  Professor Bar-Joseph is a world leader in computational methods for understanding dynamic biological systems, a critical area of research given that our cells and bodies are continuously changing.  He joined Carnegie Mellon in 2003 and played an important role in the development of the Joint Carnegie Mellon University-University of Pittsburgh Ph.D. Program in Computational Biology, which he currently directs (with James Faeder from the University of Pittsburgh).  He is well known for developing and applying rigorous computational and machine learning methods to biomedical problems.  His group has published extensively both in high impact journals and highly competitive conferences.  Their paper on sampling strategies for high throughput time series was chosen as the best paper at RECOMB 2016; RECOMB is one of the two top computational biology conferences.  In 2012, he was awarded the Overton Prize, a prize that recognizes top junior computational biologists, by the International Society for Computational Biology. He is also principal investigator (with Gregory Cooper at the University of Pittsburgh) of Big Data for Better Health, a large, multi-investigator project funded by the Pennsylvania Department of Health.</p>
					</div>
				</section>
				<section>
					<h4>The Computational Biology Department Sponsored Recent STEM Junction Symposium</h4>
					<p style="font-size:20px;">Dec 12, 2017 </p>
					<div style="width:50%; float:left;">
						<img src="images/slide7.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">The Computational Biology Department recently sponsored Fox Chapel Area High School’s STEM Junction Symposium, an all-day bioinformatics event hosting 180 students from the Pittsburgh area. The Symposium was organized by Fox Chapel senior Shania Khatri to introduce high school students with an interest in computer science and biology to the growing field of computational biology.  The Symposium is one of the first of its kind for a high school audience, but this did not dissuade Khatri. “I was convinced,’ Khatri says, ‘that there should be opportunities for high school students to explore bioinformatics.”
						The students in attendance were able to choose from multiple breakout sessions led by speakers from both Pittsburgh academia and industry.   Dr. Robert Murphy, Dr. Carl Kingsford, Dr. Phillip Compeau, and Dr. Hosein Mohimani were among the speakers for these sessions.</p>
					</div>
				</section>
				<section>
					<h4>First Year PhD student has first-authored paper accepted</h4>
					<p style="font-size:20px;">January 10, 2018</p>
					<div style="width:50%; float:left;">
						<img src="images/slide8.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Xiangrui Zeng, a first year PhD student in the Joint Carnegie Mellon-University of Pittsburgh Ph.D. program (CPCB), has been conducting a research project under the guidance of Dr. Min Xu.  His first-authored paper detailing the outcome of this project title “A convolutional autoencoder approach for mining features in cellular electron cryo-tomograms and weakly supervised coarse segmentation” has been accepted by Journal of Structural Biology.
 
						Congratulations Xiangrui on the outstanding performance!</p>
					</div>
				</section>
				<section>
					<h4>Computational Method Speeds Hunt for New Antibiotics</h4>
					<p style="font-size:20px;">Jan 22, 2018 </p>
					<div style="width:50%; float:left;">
						<img src="images/slide9.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Search Time Reduced From Hundreds of Years To Couple Of Hours

						PITTSBURGH—A team of American and Russian computer scientists has developed an algorithm that can rapidly search databases to discover novel variants of known antibiotics – a potential boon in fighting antibiotic resistance. </p>
					</div>
				</section>
				<section>
					<h4>CPCB PhD Student Recognized for Research Progress</h4>
					<p style="font-size:20px;">Jan 25, 2018</p>
					<div style="width:50%; float:left;">
						<img src="images/slide10.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Yang Yang, a third-year PhD student in the Joint Carnegie Mellon-University of Pittsburgh Ph.D. Program in Computational Biology (CPCB), joined the Computational Biology Department in January 2016 and has been working on computational genomics.  Yang received her B.E. degree in Automation and M.S. degree in Control Science and Engineering both from Tsinghua University in China. In 2015, she joined Dr. Jian Ma’s research group at the University of Illinois at Urbana-Champaign. When Dr. Ma accepted his position with Carnegie Mellon at the beginning of 2016, Yang also transferred to the University.</p>
					</div>
				</section>
				<section>
					<h4>New Carnegie Mellon Dynamic Statistical Model Follows Gene Expressions Over Time</h4>
					<p style="font-size:20px;">Jan 29, 2018</p>
					<div style="width:50%; float:left;">
						<img src="images/slide11.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Future Applications Could Range From fMRI Data to Social Networks and More
						Read Full article on CBD website! </p>
					</div>
				</section>
				<section>
					<h4>Latest update of MUMmer written by CBD Project Scientist</h4>
					<p style="font-size:20px;">Jan 31, 2018</p>
					<div style="width:50%; float:left;">
						<img src="images/slide12.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Aligning DNA sequences has been and remains one of the fundamental tool in computational biology. It is widely used to compare genomes, to find SNPs and genome variations, and as a step in many other bioinformatics pipelines.
						MUMmer, a widely used alignment package in genomics,  was originally published in 1999 when the number and the size of available genome sequences was still small. It has been updated a few times since and the third version MUMmer3 remains a very popular genome to genome aligner.
						MUMmer4 is the latest update written by Guillaume Marçais, Project Scientist in the Carnegie Mellon Computational Biology Department. It is a backward compatible version with MUMmer3, while being more versatile, being able to handle much larger sequences and faster. MUMmer4 handles genome to genome comparison as well as aligning short and long reads to a reference genome.</p>
					</div>
				</section>
				<section>
					<h4>Lane Fellow DeBlasio Publishes Book</h4>
					<p style="font-size:20px;">Feb 5, 2018</p>
					<div style="width:50%; float:left;">
						<img src="images/slide13.png" />
					</div>
					<div style="width:50%; float:right;">
						<p style="font-size:20px;">Lane Fellow Dan DeBlasio, along with his PhD advisor John Kececioglu, recently published a book titled “Parameter Advising for Multiple Sequence Alignment”, which details a method to select input-specific parameter choices for the fundamental problem of multiple sequence alignment. Using Parameter Advising can produce higher quality alignments, which leads to improvements in downstream analyses like protein structure prediction, phylogenetic inference, or homology search. The accuracy of multiple sequence alignments can be greatly impacted by choosing non-ideal parameter settings (and in turn the downstream analyses), and while the default parameter settings work well on average, they may produce poor results on some input. The book also describes Facet (short for Feature-based Accuracy Estimator) which when used with Parameter Advising can automatically select improved parameters for any input set of protein sequences. “Parameter Advising for Multiple Sequence Alignment” is part of Springer International’s Computational Biology Series.</p>
					</div>
				</section>								
			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>
			// More info about config & dependencies:
			// - https://github.com/hakimel/reveal.js#configuration
			// - https://github.com/hakimel/reveal.js#dependencies
			Reveal.initialize({
				dependencies: [
					{ src: 'plugin/markdown/marked.js' },
					{ src: 'plugin/markdown/markdown.js' },
					{ src: 'plugin/notes/notes.js', async: true },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } }
				]
			});
		</script>
	</body>
</html>
